tests/contenttypes_tests/test_checks.py

```diff
@@ -216,3 +217,24 @@ class InvalidBookmark(models.Model):
                 id='fields.E001',
             )
         ])
+
+
+class ModelCheckTests(SimpleTestCase):
```

Comment:
You probably need to pass the isolated app configs when calling `check_model_name_lengths`. See [`isolate_apps`](https://docs.djangoproject.com/en/1.11/internals/contributing/writing-code/unit-tests/#django.test.utils.isolate_apps) docs about it. I'd suggest you use `attr_name='apps'` and pass `app_configs=self.apps.get_app_configs()` to `check_model_name_lengths`.